Tests/ProcessLib/TestParameter.cpp

```diff
@@ -0,0 +1,176 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include <gtest/gtest.h>
+#include <logog/include/logog.hpp>
+
+#include <boost/property_tree/xml_parser.hpp>
+#include <numeric>
+#include <sstream>
+#include <vector>
+
+#include "BaseLib/ConfigTree.h"
+#include "Tests/TestTools.h"
+
+#include "MeshLib/Mesh.h"
+#include "MeshLib/PropertyVector.h"
+#include "MeshLib/MeshGenerators/MeshGenerator.h"
+
+#include "ProcessLib/Parameter/PropertyIndexParameter.h"
+
+// make useful line numbers appear in the output of gtest
```

Comment:
The code from here through line 95 (end of anonymous namespace) is only needed for testing ConfigTree. It can be omitted from this unit test.
A ConfigTree can be manually created via
```
BaseLib::ConfigTree config(ptree, "", BaseLib::ConfigTree::onerror, BaseLib::ConfigTree::onwarning);
```